Library/Formula/gnu-apl.rb

```diff
@@ -14,12 +14,11 @@ class GnuApl < Formula
   # GNU Readline is required; libedit won't work.
   depends_on "readline"
   depends_on :macos => :mavericks
+  depends_on "postgresql" => :recommended
```

Comment:
I think it also applies to 1.4 because it is probably required to use sql. I will change it to `:optional`.